kedro/__init__.py

```diff
@@ -8,4 +8,10 @@
 
 import logging
 
+try:
+    from IPython import get_ipython
+    from .extras.extensions.ipython import load_ipython_extension
+except ModuleNotFoundError:
+    pass  # Skip it when IPython is not instsalled
+
```

Comment:
This also works fine, but as a consequence, these configurations will be broken. Since everything is under `kedro` namespace and effectively they all depend on each other.


```
[[tool.importlinter.contracts]]
name = "CLI > Context > Library, Runner > Extras > IO & Pipeline"
type = "layers"
```